integration_tests/common.rs

```diff
@@ -53,6 +53,22 @@ impl fmt::Display for MerchantServices {
     }
 }
 
+/// Options for who is being simulated as the malicious party (for use in DB store/restore)
+#[derive(Debug, Clone, Copy, EnumIter)]
+pub enum MaliciousParty {
+    Customer,
+    Merchant,
+}
+
+impl fmt::Display for MaliciousParty {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match *self {
+            MaliciousParty::Customer => write!(f, "customer"),
+            MaliciousParty::Merchant => write!(f, "merchant"),
```

Comment:
note: You can simplify using this syntax, which defaults to using the name of the enum variant: https://stackoverflow.com/questions/32710187/how-do-i-get-an-enum-as-a-string#32712140

(This is not relevant if you end up taking out this enum, but FYI).

Reply:
Ah, I was worried about cases haha, but I'll take this enum out anyway